backend/editor/graph_db.py

```diff
@@ -3,34 +3,41 @@
 """
 import contextlib
 import contextvars  # Used for creation of context vars
+import logging
 
-from neo4j import GraphDatabase  # Interface with Neo4J
+import neo4j  # Interface with Neo4J
 
 from . import settings  # Neo4J settings
 from .exceptions import SessionMissingError  # Custom exceptions
 from .exceptions import TransactionMissingError
 
+log = logging.getLogger(__name__)
```

Comment:
I think we don't use this anywhere in this file, so I think this line and the imports can be removed

Reply:
Yep, but we should log more (at least debug or info) so I think we can keep it (it's a good pattern to have it)